docs/docs/en/guides/features/previews.md

```diff
@@ -109,6 +109,76 @@ Testing new functionality should be a part of any code review. But having to bui
 Once your Tuist project is connected with your Git platform such as [GitHub](https://github.com), add a <LocalizedLink href="/cli/share">`tuist share MyApp`</LocalizedLink> to your CI workflow. Tuist will then post a Preview link directly in your pull requests:
 ![GitHub app comment with a Tuist Preview link](/images/guides/features/github-app-with-preview.png)
 
+
+## In-app update notifications {#in-app-update-notifications}
+
+The [TuistSDK](https://github.com/tuist/sdk) enables your app to detect when a newer preview version is available and notify users. This is useful for keeping testers on the latest build.
```

Comment:
Do we want to use `TuistSDK` as a name? I was thinking we can refer to it as the "Tuist SDK" in the docs, even if the name of the library product is `TuistSDK`.

Reply:
I can change it to Tuist SDK, yeah 👍